src/ctapipe/io/tests/test_table_loader.py

```diff
@@ -369,6 +369,19 @@ def test_read_shower_distributions(dl2_merged_file):
         assert np.all(histograms["histogram"].sum(axis=(1, 2)) == [2000, 1000])
 
 
+def test_read_scheduling_information(dl2_merged_file):
+    from ctapipe.io import TableLoader
+
+    with TableLoader(dl2_merged_file) as table_loader:
+        scheduling = table_loader.read_scheduling_information()
+        assert len(scheduling) == 2
+        assert np.all(scheduling["sb_id"] == [4, 1])
```

Comment:
For better error messages, use `np.testing.assert_array_equal(scheduling["sb_id"], [4, 1])`

Reply:
Done